draft-scenarios/coops/close_to_enemies.tex

```diff
@@ -0,0 +1,220 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Cooperative Scenario}{Close to enemies}{\images/balistic.png}
+
+\begin{multicols}{2}
+
+\textbf{Author:} Kelghor
+
+\textbf{Version:} 1.5
+
+\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1313651865648369725}{Archon Studio Discord}
+
+\textit{"Sir! They're here again!"}
+
+\textit{"Sound the alarm, we'll repel them as always!" you command, and go to defend the city. Just as you have done many times before. Such is the hard life of the surrounding clans, with constant wars and skirmishes with nearby neighbors. Fortunately, your alliance is strong, so you know your back is covered.}
+
+\textit{Therefore, the time has come to end the long-standing wars and crush your enemies like a bug once and for all!}
+
+\subsection*{\MakeUppercase{Scenario Length}}
+
+This Scenario is played over 7 Rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+
+\textbf{Player Count:} 1-6
+
+\textbf{Starting Resources:} 14 \svg{gold}, 5 \svg{building_materials}, 2 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 0 \svg{valuables}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item  1 × A Few \svgunit{bronze} Units of your choice
+\end{itemize}
+
+\textbf{Town Buildings:} \svgunit{bronze} Dwelling.
+
+\textbf{Additional Bonus:}: Only base bonus by difficulty (see the main rules).
+
+\textbf{Map Tile Pool:} Take $P$ × random Far (II–III) map Tiles as shared pool. None of them must contain a settlement. ($P$ stands for the number of players)
+
+\subsection*{\MakeUppercase{Map Setup}}
+
+Take the following Map Tiles and arrange them as shown in the Scenario map layout.
+Setup Far (II--III) Map Tiles with settlements of same Faction as players Towns as show a color Cube, or see example.
```

Comment:
```suggestion
Take the following Map Tiles and arrange them as shown in the Scenario map layout.
Place Far (II--III) Map Tiles with Settlements that match your Faction's Towns. Mark these with a colored Cube (see example).
```
Is that what you meant to say here?

Reply:
BTW, those cubes are barely visible. Maybe you could highlight those tiles on the picture with matching colors?

Reply:
Yes i will do it after code revision :-)